lib/rangeclient.rb

```diff
@@ -37,6 +40,7 @@ def expand(arg)
     http.use_ssl = @ssl
     req = Net::HTTP::Get.new('/range/list?' + escaped_arg)
     resp = http.request(req)
+    raise QueryException.new(resp.body) unless resp.code == "200"
```

Comment:
nit: more canonical way to do this is `Net::HTTPSuccess === resp` (or `resp.is_a?(Net::HTTPSuccess)` .... they're effectively equivalent), which avoids the ugly 200-is-a-string issue.